Resnet-18 and VGG 11 implementation in E2CNN

namankhetan opened the pull request.

namankhetan commented:
#46 This the issue for which this pull request is being raised.

Gabri95 commented:
Hi @namankhetan 

Thanks for creating this pull request!

Unfortunately, I can not merge it yet since it does not pass all the checks (have a look [here](https://github.com/QUVA-Lab/e2cnn/pull/46/checks)).
Could you fix the signature in the commits?

Also, before including these examples in the library, I'd like to check the models work well.
Have you already tried to train these models on some dataset? Do you have some results I can try to replicate?

Best,
Gabriele

namankhetan commented:
-Hii @Gabri95 Thanks for appreciating my work in the issue.
 I fixed all the checks :),
The models are working fine, I have added test cases at the bottom of the codes. I have also tested the models on ROT-MNIST dataset, they are acheiving accuracy around 98.5-99% ,i haven't done much hyperparameter tuning as it's sufficent to see the correctness of the models.
Also as in the issue #45 @mkool67 mentioned that the models worked for him/her. 

Thanks.
